src/repo.rs

```diff
@@ -2051,6 +2052,54 @@ impl Repository {
         }
         Ok(ignored == 1)
     }
+
+    /// Initializes a rebase operation to rebase the changes in `from_branch`
+    /// relative to `upstream` onto '`to_branch`. To begin the rebase
+    /// process, call `next()` or `operations()` on the returned object.
+    pub fn rebase_init(&self,
+                       from_branch: Option<AnnotatedCommit>,
+                       upstream: Option<AnnotatedCommit>,
+                       to_branch: Option<AnnotatedCommit>,
```

Comment:
Since this doesn't need ownership, could these be `&AnnotatedCommit`?

Reply:
Done.